native-image/graalvm-native-image-quick-start/README.md

```diff
@@ -354,7 +358,7 @@ when your application runs.
 >Note: It is important to exercise all the code paths in your application when running the tracing agent, in order 
 to ensure that all cases of reflection are discovered.
 
-For more information about the tracing agent, see [Assisted Configuration with Tracing Agent](https://www.graalvm.org/reference-manual/native-image/Agent/).
+For more information about the tracing agent, see [Assisted Configuration with Tracing Agent](https://www.graalvm.org/22.0/reference-manual/native-image/Agent/).
```

Comment:
The original link gave a 404: https://www.graalvm.org/reference-manual/native-image/Agent/
Seems to require the version: https://www.graalvm.org/22.0/reference-manual/native-image/Agent/